ros2pkg/ros2pkg/resource/cmake/Config.cmake.in.em

```diff
@@ -0,0 +1,16 @@
+# - Config file for the @(project_name) package
+# It defines the following variables
+#  @(project_name)_INCLUDE_DIRS - include directories for FooBar
+#  @(project_name)_LIBRARIES    - libraries to link against
+#  @(project_name)_EXECUTABLE   - the bar executable
+ 
+# Compute paths
+get_filename_component(@(project_name)_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
```

Comment:
No need to create your own variable for this. `@(PROJECT_NAME)_DIR` is defined automatically pointing to the directory containing the CMake config file of the package.